src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -740,6 +742,7 @@ public static IEnumerable<InstructionSetInfo> ArchitectureToValidInstructionSets
                     yield return new InstructionSetInfo("Vector64", "", InstructionSet.ARM64_Vector64, false);
                     yield return new InstructionSetInfo("Vector128", "", InstructionSet.ARM64_Vector128, false);
                     yield return new InstructionSetInfo("Dczva", "", InstructionSet.ARM64_Dczva, false);
+                    yield return new InstructionSetInfo("rcpc", "", InstructionSet.ARM64_Rcpc, true);
```

Comment:
```suggestion
                    yield return new InstructionSetInfo("rcpc", "Rcpc", InstructionSet.ARM64_Rcpc, true);
```

Going by the code and exception; with `specifiable: true`, we add the `ManagedName`.

Reply:
Thanks!